focal-rt-ros2/rootfs/setup/phase1.sh

```diff
@@ -24,18 +50,13 @@ systemctl enable cpu-frequency
 
 # TODO: If specified, create an image with isolcpus already setup.
 
-export DEBIAN_FRONTEND=noninteractive
-
-# Remove some packages that are likely not needed:
-# - snapd: no one packages their robot apps with snap, right?
-# - fwupd: I don't think we need to update devices firmware like a logitech mouse, and it also uses like 20MB of RAM...
-apt-get purge --autoremove -y snapd fwupd
-
-# Install ROS2
+# Setup ROS distro without installing ROS to save space on the image.
 sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key  -o /usr/share/keyrings/ros-archive-keyring.gpg
-
-sudo apt-get update
-sudo apt-get install -y ros-galactic-ros-base
+# sudo apt-get install -y ros-galactic-ros-base
```

Comment:
```suggestion
sudo apt-get install -y ros-galactic-ros-base
```

With ROS 2 installed, do we increase the image size a lot?

Reply:
```
-rw-r--r-- 1 root root  1.4G Feb  6 15:56 ubuntu-20.04.3-rt-ros2-galactic-arm64+raspi.img.gz
-rw-r--r-- 1 root root 1015M Feb  6 02:02 ubuntu-20.04.3-rt-ros-arm64+raspi.img.gz
```

~400MB after compression. 1.4GB should still be good, as GH releases limit is [2GB](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases#storage-and-bandwidth-quotas). I'll reinstate this.

Reply:
Thank you!